### PR TITLE
Polymer 3 TS Class: ServerCard

### PR DESCRIPTION
**Pull request by daniellacosse**
--> Blocks https://github.com/Jigsaw-Code/outline-client/pull/1106

### CODE REVIEW

File: src/www/ui_components/server_card.ts [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+      this.fireShowServerRename();
+    }
+
+    // This can leave the pressed paper-item in the selected state,
+    // causing it to get selected styling (e.g. font-weight: bold),
+    // so explicitly deselect it:
+    this.async(() => (this.$.menu as HTMLInputElement).select());
+  }
+
+  private fireConnectPressed() {
```
Comment (fortuna) [inline src/www/ui_components/server_card.ts:269]: These functions only add indirection, hurting readability. I'd inline them instead.

**Review by fortuna — APPROVED** [review]

File: src/www/ui_components/server_card.ts (hunk 1 above)
Comment (daniellacosse) [reply src/www/ui_components/server_card.ts:269]: Totally, as long as we're consistent one way or the other - before we had a combination of inlining and separate functions

**Review by fortuna — APPROVED** [review]

File: src/www/ui_components/app-root.js [hunk 2]
```diff
@@ -47,6 +47,7 @@ import './outline-icons.js';
 import './privacy-view.js';
 import './server-connection-viz.js';
 import './server_list.ts';
+import './server_card.ts';
```
Comment (bemasc) [inline src/www/ui_components/app-root.js:50]: Why are these files using underscores?  I think everything else is using hyphens.

File: src/www/ui_components/server_card.ts [hunk 3]
```diff
@@ -0,0 +1,282 @@
...
+  }
+
+  protected onMenuItemPressed({detail: {selected}}: CustomEvent) {
+    const {serverName, serverId} = this;
+
+    if (selected === 'forget') {
+      this.fire('ForgetPressed', {serverId});
+    } else if (selected === 'rename') {
+      this.fire('ShowServerRename', {serverName, serverId});
+    }
```
Comment (bemasc) [inline src/www/ui_components/server_card.ts:275]: Else some kind of warning?

**Review by bemasc — APPROVED** [review]

File: src/www/ui_components/app-root.js (hunk 2 above)
Comment (daniellacosse) [reply src/www/ui_components/app-root.js:50]: Our style guide wants underscores

**Pull request merged**